recipes/r-ngsplot-hg19/meta.yaml

```diff
@@ -0,0 +1,21 @@
+{% set version = "3.00" %}
+{% set name = "ngsplotdb-hg19" %}
+
+package:
+  name: 'ngsplotdb-{{ name|lower }}'
+  version: '{{ version }}'
+  md5: 172a6a93de3d1ae8a3ad6d6d79e911f7
+build:
+  number: 0
+requirements:
+  run:
+    - 'r-ngsplot'
+    - r-base
+    - wget
+test:
+  commands:
+    - ngsplotdb.py install "${TARBALL}" > "${PREFIX}/.messages.txt" 2>&1
```

Comment:
The test is run after the package is installed, i.e., after the `post-link.sh` is run.
So, that `ngsplotdb.py install` is already done at that point. Rather than executing it again, you should run a test that checks if the database has been successfully installed (either with some (short-running!) command from `r-gnsplot` or if that's not available, at least something like `test -f "${PREFIX}/some-file-path"`.)